doc/usecases/layers/ph_values_layer.md

```diff
@@ -0,0 +1,24 @@
+# Use Case: pH Values Layer
+
+## Summary
+
+- **Scope:** pH Values Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can add and edit pH values in their plan using the pH values layer
```

Comment:
This is also in zones: drawing with big brush to indicate where the pH value is 0..14.